cohort_4/playbook/ACTIVITIES.md

```diff
@@ -88,6 +89,109 @@
 
 <!-- markdown-toc end -->
 
+# Creating a safe space
+
+The one rule that we have at Hack Camp is that we make it a safe space.
+We all know how bad it feels when we are made fun of or when someone makes
+us feel uncomfortable.
+
+Below are some scenarios which may occur and how to handle them:
+
+## Someone writes or says "X person is gay." in a playful joking manner
+
+Although in many cases this is intended as a joke, but it sets the norm that
+at Hack Camp
+
+- being gay is bad
+- if you are gay, you should feel bad
+- gay people are not welcome here
+
+This is not an environment that we want to create. This needs to be addressed
+quickly and in a very serious tone to make sure everyone understands that this
+is the one thing that is extremely important to us.
+
+However at the same time, we don't want to make the person who said it feel
+really bad and feel shameful because that could hurt just as much. They thought
+that it was just a joke.
+
+Here's how I might handle the situation. Note that this is just one way to
+handle the situation, feel free to handle it in a way that you choose as well.
+
+Also note that this is only a suggestion for what you might bring up. Feel free
+to say as much or as little of it based on what you feel is necessary and what
+is comfortable for you.
+
+- Can I get everyone's attention please
+- I noticed earlier that someone posted the phrase say "X person is Gay"
```

Comment:
I think that because this is referencing a specific instance that we know can occur, the incorrect grammar of the quote is fine.